### DISCUSSION TITLE
ㅠㅠ아래 알려주신 수정사항 사용 방법 여쭙니다.

### DISCUSSION THREAD

**Discussion by sdr03095** [changdoc/naver-vaccine-macro#261] [Q&A]
코드 zip 파일을 다운받았는데, 후에 어떻게 해야 할까요ㅠㅠ?

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/issues/29

**Comment by changdoc** [comment]
끝에 보시면 


크롬 스토어에서 받은 확장 프로그램은 삭제하고 진행해주세요.
다운 받은 최신 코드, 혹은 특정 버전 zip 파일 위치를 기억해두고 아래 블로그 설명 대로 따라하시면 됩니다.
(크롬 개발자 모드 관련 설명 블로그) https://trend21c.tistory.com/1030

**Reply by sdr03095** [reply]
감사합니다!!!

**Reply by sdr03095** [reply]
한가지 더 여쭈어도 될까요?
아래 그림처럼 떴는데, 화이자만 노릴 수 있을까요..!?
<img width="305" alt="스크린샷 2021-08-12 오후 12 14 37" src="https://user-images.githubusercontent.com/88127573/129132946-07a8cdc4-75e6-4104-8ada-14d7a5242942.png">

**Reply by changdoc** [reply]
매크로 창에서
![image](https://user-images.githubusercontent.com/1879494/129133499-25946bae-e5a7-4b7e-b8ad-bdc7d31a4c3f.png)
화이자 선택 해두시면 화이자 잔여 수량이 발생할 경우에 예약 신청을 수행합니다.

**Reply by sdr03095** [reply]
이 창이 확장프로그램을 지우니 없어졌어요ㅠㅠ...!

**Reply by sdr03095** [reply]
다시 크롬확장프로그램 다운 후 하고 있씁니다. 감사합니다.
